.github/workflows/manager_docker_images.yml

```diff
@@ -5,17 +5,15 @@ on:
     branches:
       - main
     tags:
-      - v*
+      - manager-v*
```

Comment:
we should use the same tag for both images, just push a `v*` tag and both images must be build